app/Models/Company.php

```diff
@@ -261,7 +261,12 @@ public function setupDefaultData()
 
         return true;
     }
-
+    protected function getDefaultCurrencyId()
+    {
+        $currencyCode = env('CRATER_DEFAULT_CURRENCY');
+        $currency = Currency::where('code', $currencyCode)->first();
+        return $currency ? $currency->id : 13; // Default to 13 (INR)if currency not found
```

Comment:
@deepthi-mantena Please add a log messgae before return on what is the currency is being set

Reply:
@mohan-13 Updated the PR .Please verify